package.json

```diff
@@ -27,7 +27,8 @@
     "passport-local": "^1.0.0",
     "request": "^2.87.0",
     "winston": "^3.0.0",
-    "winston-syslog": "^2.0.0"
+    "winston-syslog": "^2.0.0",
+    "ini": "^1.3.5"
```

Comment:
I think you should add this dependency in the commit it's used.

Reply:
Done!